.github/workflows/ci.yml

```
@@ -16,19 +16,24 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        ruby: [2.3.x, 2.4.x, 2.5.x, 2.6.x]
+        ruby: [1.9, 2.0, 2.1, 2.2, 2.3, 2.4, 2.5, 2.6]
 
     steps:
     - uses: actions/checkout@v1
 
     - name: Install dependencies
       run: |
         sudo apt-get install libcurl4-openssl-dev
-
+    
+    - name: Set up RVM
+      run: |
+        curl -sSL https://get.rvm.io | bash
+        
     - name: Set up Ruby
-      uses: actions/setup-ruby@v1
-      with:
-        ruby-version: ${{ matrix.ruby }}
+      run: |
+        source $HOME/.rvm/scripts/rvm
+        rvm install ${{ matrix.ruby }} --binary
```

```suggestion
        rvm install ${{ matrix.ruby }} --disable-binary
```

The build failures seem to be about there being no prepared binaries.